Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -333,6 +333,14 @@ public class XPCServer {
 	internal func acceptMessage(connection: xpc_connection_t, message: xpc_object_t) -> Bool {
 		fatalError("Abstract Method")
 	}
+
+    public var serviceName: String? {
```

Comment:
Looks like `XPCMachServer` doesn't override this.

Separately is this useful for an API user to know? Excluding anonymous servers there's likely to only ever be one server running, and anonymous servers don't have a service name.

Reply:
I use it in my logger in my project.

Reply:
Is logging that any more useful than logging the bundle id?

Reply:
Nah, but that's kind of an implementation detail. The "service name" is what I'm looking to log, and the bundle ID just happens to be the source for it at the moment.